core/src/Kokkos_View.hpp

```diff
@@ -560,7 +560,10 @@ class View : public Impl::BasicViewFromTraits<DataType, Properties...>::type {
       : base_t(
             arg_prop,
             Impl::mapping_from_array_layout<typename mdspan_type::mapping_type>(
-                arg_layout)) {}
+                arg_layout)) {
```

Comment:
This is a little weird, is your branch up to date?

Reply:
Literally a fresh fork, but must have messed something while doing the whole "sign off" thing.